components/dribbble/sources/new-shot-created/new-shot-created.mjs

```diff
@@ -0,0 +1,33 @@
+import dribbble from "../../dribbble.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  key: "dribbble-new-shot-created",
+  name: "New Shot Created",
+  description: "Emit new events when new shots are created",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    dribbble,
+    timer: {
+      type: "$.interface.timer",
+      label: "Polling schedule",
+      description: "How often to poll the Dribble API",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+  },
+  async run() {
+    const shots = await this.dribbble.getShots();
+
+    for (const shot of shots) {
+      this.$emit(shot, {
+        id: shot.id,
+        summary: shot.title,
+        ts: Date.parse(shot.published_at),
+      });
+    }
```

Comment:
I think you could check `published_at` to make sure the `shot` is new, since `dedupe` only works for a limited number of items.
```suggestion
    for (const shot of shots) {
      if (Date.parse(shot.published_at) > lastTimestamp) {
        this.$emit(shot, {
          id: shot.id,
          summary: shot.title,
          ts: Date.parse(shot.published_at),
        });
      }
    }
```